packages/core/content-type-builder/admin/src/components/IconPicker/index.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useRef } from 'react';
+import { Box, Flex, Icon, Typography, Searchbar, IconButton } from '@strapi/design-system';
+import * as Icons from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import { inputFocusStyle } from '@strapi/design-system';
+import styled from 'styled-components';
+import { getTrad } from '../../utils';
+
+const IconPickerWrapper = styled(Flex)`
+  position: relative;
+
+  label {
+    border-radius: 4px;
+    ${inputFocusStyle}
+  }
+
+  input {
+    position: absolute;
+    opacity: 0;
+  }
+`;
+
+const IconPick = ({ iconKey, name, onChange, isSelected }) => {
+  return (
+    <label htmlFor={iconKey}>
```

Comment:
The label doesn't have any text a screenreader could use to read what this is to a user. Could you please introduce one?